rollup.config.mjs

```diff
@@ -72,6 +73,7 @@ export default [{
         gzipSize: true,
         open: true,
       }),
+    json()
```

Comment:
Could we generate ts files instead of json to avoid the added complexity ?

Reply:
@chambo-e Sure!